wisembly/amqp-bundle/1.4/config/packages/wisembly_amqp.yaml

```diff
@@ -0,0 +1,16 @@
+wisembly_amqp:
+    console_path: '%kernel.project_dir%/bin/console'
+    broker: 'Wisembly\AmqpBundle\Broker\PhpAmqpLibBroker'
```

Comment:
The only change from `1.3` is this, as using the alias `oldsound` is deprecated.